desktop/models/sale.coffee

```diff
@@ -39,6 +39,8 @@ module.exports = class Sale extends Backbone.Model
 
   bidUrl: (artwork) ->
     "#{@href()}/bid/#{artwork.id}"
+  
+  predictionUrl: -> "#{PREDICTION_URL}/#{@get('slug')}"
```

Comment:
updated in my latest commit with this suggestion, plus a new match on the error message from artsy/gravity#10972